cmd/kubeapps-apis/plugins/fluxv2/packages/v1alpha1/repo.go

```diff
@@ -45,6 +45,7 @@ const (
 var (
 	// default poll interval is 10 min
 	defaultPollInterval = metav1.Duration{Duration: 10 * time.Minute}
+	redactedString      = "REDACTED"
```

Comment:
This should be a `const` rather than a package-global variable shouldn't it?

Reply:
👍